app/cafe.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
```

Comment:
You're comparing a variable to `None` here, which isn't necessary. According to the checklist, you can simply use `if visitor.get('vaccine'):` instead of `if not visitor.get('vaccine'):`.